samples/file-encryptor/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ project("File Encryptor Sample" LANGUAGES C CXX)
 find_package(OpenEnclave CONFIG REQUIRED)
 
 set(CMAKE_CXX_STANDARD 11)
+set(OE_CRYPTO_LIB openssl)
```

Comment:
@mingweishih @radhikaj  Do we want to default to openssl now given that it's still gated as experimental? There's also a question of timing this PR since the modifications to the sample are not gated by the use of the `BUILD_OPENSSL` flag currently.

Reply:
No, we should not default to using openssl. We will only merge this PR once we drop the experimental flag and ship the openssl libs with OE package.

Reply:
Yes, I agree. Maybe we should wait until the OpenSSL become non-experimental so that we can merge this PR.